README.md

```diff
@@ -20,11 +20,11 @@ Go to [excalidraw.com](https://excalidraw.com) to start sketching.
 
 Read the latest news and updates on our [blog](https://blog.excalidraw.com). A good start is to see all the updates of [One Year of Excalidraw](https://blog.excalidraw.com/one-year-of-excalidraw/).
 
-## We accept donations
+## Supporting Excalidraw
```

Comment:
I like the more direct approach... /cc @vjeux?

Reply:
I feel this is a better way of communicating to viewers that we need their support :)

Reply:
@ad1992 which one do you mean/prefer?

Reply:
The new one `Supporting Excalidraw`